.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -0,0 +1,62 @@
+name: 🐛 Bug Report
+description: File a bug report
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for filing a bug report! This issue tracker is for [GitHub Desktop](https://desktop.github.com). If you are experiencing issues with the Linux fork of GitHub Desktop please open an issue [in that repository](https://github.com/shiftkey/desktop). If you are experiencing issues with github.com please [contact GitHub Support](https://support.github.com/).
```

```suggestion
        Thanks for filing a bug report! This issue tracker is for [GitHub Desktop](https://desktop.github.com). If you are experiencing issues with the Linux fork of GitHub Desktop please open an issue [in this repository](https://github.com/shiftkey/desktop). If you are experiencing issues with github.com please [contact GitHub Support](https://support.github.com/).
```

Update grammer!

---

I think this makes this slightly less clear - maybe something like this?
```suggestion
        Thanks for filing a bug report! This issue tracker is for [GitHub Desktop](https://desktop.github.com). If you are experiencing issues with the Linux fork of GitHub Desktop please open an issue [in its repository](https://github.com/shiftkey/desktop). If you are experiencing issues with github.com please [contact GitHub Support](https://support.github.com/).
```

---

Yup, either way works. Thanks for the alternative suggestion @Rexogamer